docs/charts/42/delta/supplemental-data.html

```diff
@@ -179,13 +179,22 @@
 	<tr><td class='source'>metric-ton-systems</td><td class='target'>metric</td><td class='target'>▷removed◁</td></tr>
 	<tr><td class='source'>quarter-systems</td><td class='target'>▷missing◁</td><td class='target'>metric</td></tr>
 	<tr><td class='source'>tonne-systems</td><td class='target'>▷missing◁</td><td class='target'>metric</td></tr>
-	<tr><td class='source' rowSpan='7'><a name='Likely' href='#Likely'>Likely</a></td><td class='source' rowSpan='7'><a name='Subtag' href='#Subtag'>Subtag</a></td><td class='source'>ajt</td><td class='target'>▷missing◁</td><td class='target'>ajt_Arab_TN</td></tr>
+	<tr><td class='source' rowSpan='16'><a name='Likely' href='#Likely'>Likely</a></td><td class='source' rowSpan='16'><a name='Subtag' href='#Subtag'>Subtag</a></td><td class='source'>ajt</td><td class='target'>▷missing◁</td><td class='target'>ajt_Arab_TN</td></tr>
 	<tr><td class='source'>ann</td><td class='target'>…</td><td class='target'>ann_Latn_NG</td></tr>
 	<tr><td class='source'>oc_ES</td><td class='target'>▷missing◁</td><td class='target'>oc_Latn_ES</td></tr>
 	<tr><td class='source'>pis</td><td class='target'>▷missing◁</td><td class='target'>pis_Latn_SB</td></tr>
 	<tr><td class='source'>smd</td><td class='target'>▷missing◁</td><td class='target'>smd_Latn_AO</td></tr>
 	<tr><td class='source'>snb</td><td class='target'>▷missing◁</td><td class='target'>snb_Latn_MY</td></tr>
 	<tr><td class='source'>tok</td><td class='target'>▷missing◁</td><td class='target'>tok_Latn_001</td></tr>
+	<tr><td class='source'>und_Laoo_AU</td><td class='target'>▷missing◁</td><td class='target'>hnj_Laoo_AU</td></tr>
```

Comment:
This need to be updated, running after the und_Laoo_* removal from the likely subtags.

Reply:
updated